src/EmbargoesEmbargoesService.php

```diff
@@ -167,12 +167,12 @@ public function getMediaParentNids($mid) {
   public function getParentNidsOfFileEntity($file) {
     $relationships = file_get_file_references($file);
     if (!$relationships) {
-      $nids = NULL;
+      $nids = [];
     }
     else {
       foreach ($relationships as $relationship) {
         if (!$relationship) {
-          $nids = NULL;
+          $nids = [];
```

Comment:
Seems like some weird stuff going on in this function in general... should it be setting `$nids` as en empty array at the start of the function, and then merging things in?... 'cause if there happened to be multiple relationships, whatever the last one sets would win? Possibly an empty array? Dunno... seems spooky?